tests/integration/devfile/cmd_devfile_push_test.go

```diff
@@ -17,32 +17,46 @@ var _ = Describe("odo devfile push command tests", func() {
 	var namespace, context, cmpName, currentWorkingDirectory, projectDirPath string
 	var projectDir = "/projectDir"
 	var sourcePath = "/projects/nodejs-web-app"
+	var clusterType helper.CliRunner
 
-	// TODO: all oc commands in all devfile related test should get replaced by kubectl
-	// TODO: to goal is not to use "oc"
-	oc := helper.NewOcRunner("oc")
+	// Using program commmand according to cluter type in devfile
+	if os.Getenv("KUBERNETES") == "true" {
+		clusterType = helper.NewKubectlRunner("kubectl")
```

Comment:
Can we have a better variable name for this? clusterType doesn’t seem appropriate for a runner.

Reply:
> Can we have a better variable name for this? clusterType doesn’t seem appropriate for a runner.

@girishramnani I am open to a better name. Can you please suggest one?

Reply:
done